administrator-guides/federation/README.md

```diff
@@ -0,0 +1,88 @@
+# Federation
+
+## Introduction
+
+Federation allows communication with other Rocket.Chat servers.
+
+When you open Federation's config screen, you will see this:
+
+![Initial Federation Screen](./initial.png){:style="width='100%' height='auto'"}
+
+Below, a brief explanation of each configuration option:
+
```

Comment:
Need a high level description of *what* is implemented in this version, what should users be expecting to be working.   And what are the known restrictions.  What will be coming in future versions.